frontend-react/src/hooks/UseLookupTable.ts

```diff
@@ -123,21 +123,21 @@ export const getSenderAutomationDataRows = async <T>(
 };
 
 const useLookupTable = <T>(
-    tableName: LookupTables,
-    dataSetName: string | null = null,
+    dataSetName: string | LookupTables = LookupTables.VALUE_SET,
```

Comment:
This looks to be the only use of this enum anymore (that and the comparison on line 134 below).
@sethdarragile6 @doug-s-nava Waited to get input from y'all: is this enum something we'd intended to disappear after this change? If so I'll take it out and we can use a single string default for `dataSetName` here and on line 134 in the comparison.

Reply:
It is also still used in tests it appears

Reply:
yeah, the enum would still be in use. What do we gain by eliminating it? Not exactly clear on the goal of this PR, beyond the simplification of not needing to default to null (which will go away)

Reply:
The original params required both a `LookupTables` enum value AND the data set name -- then, its only use for `dataSetName` was to measure if it was null or not, which it was by default, meaning that it'd pass along the `tableName` value to the network calls. By passing setting this to default to the enum value, we're saying "if they don't provide `dataSetName` then this is a call for `getSenderAutomationData`. If they DO provide one (i.e. it isn't the default) then it's for `getSenderAutomationDataRows`

Reply:
I see now why this enum may no longer be useful. I think you can remove it and just hardcode the value of  LookupTables.VALUE_SET where necessary

Reply:
Just my $.02 here. It might be better form to just change the values in the LookupTables enum to match what we expect on the backend, rather than eliminating it. Just thinking of the fact that that table name is coming from the URL, which one can manually change to whatever crazy thing you can dream up. And if we're not already doing it, it could be a quick win to validate that against the enum before it's even passed on to the API call.

Reply:
@sethdarragile6 Changes like that sound like something that'll be overwritten by Doug's react-query refactor of this part of the app